src/github.com/stellar/go-horizon/resource_account.go

```diff
@@ -82,3 +129,10 @@ func AmountToString(amount int64) string {
 	frac := amount % stellarbase.One
 	return fmt.Sprintf("%d.%07d", whole, frac)
 }
+
+func IsTrue(value int32) bool {
```

Comment:
It seems to me this method is not very useful.  The problem is that for it to be useful you still have to do some bitmath prior to calling these method (in the way it is used).  Better, IMO, would be to provide the "IsFlagSet" func:

```go
func IsFlagSet(flags int, mask int) bool {
  return (flags & mask) != 0;
}
```

The callsite then looks much cleaner, IMO:

```go
hasAuthRequired := IsFlagSet(ac.Flags, db.FlagAuthRequired)
```